	To apply this patch, from root of Monetdb source tree,
	execute the following commands:

		$ cd clients/src/python/monetdb/sql
		$ patch -p0 < 3.monetdb.sql.converters.diff

--- /home/mark/old.py	2009-12-21 09:18:54.000000000 -0500
+++ ./converters.py	2009-12-21 09:18:07.000000000 -0500
@@ -118,6 +118,7 @@
         #data = data.replace("\\\\", "\\")
         data = data.replace("\'", "'")
         data = data.replace("\\\"", "\"")
+	data = data.decode("utf8")
         return data[1:-1]
 
     def __decimal(self, data):
